### DISCUSSION TITLE
6.1.0

### DISCUSSION THREAD

**Discussion by DCSBL** [DCSBL/python-frank-energie#177] [General]
# [6.1.0](https://github.com/DCSBL/python-frank-energie/releases/tag/6.1.0)

🎉 Add support for SmartBatteries, thanks @berendhaan!

## What's Changed
* Fix datetime.utcnow deprecation by @berendhaan in https://github.com/DCSBL/python-frank-energie/pull/165
* Add listing for SmartBatteries and related tests by @berendhaan in https://github.com/DCSBL/python-frank-energie/pull/176
* Update GraphQL query for smartBatteries query by @berendhaan in https://github.com/DCSBL/python-frank-energie/pull/178
* Add battery session data and corresponding tests by @berendhaan in https://github.com/DCSBL/python-frank-energie/pull/189

## Dependencies
* Bump pytest-asyncio from 0.23.4 to 0.23.5 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/147
* Bump python-dateutil from 2.8.2 to 2.9.0.post0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/148
* Bump pyupgrade from 3.15.0 to 3.15.1 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/149
* Bump pylint from 3.0.3 to 3.1.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/150
* Bump pre-commit from 3.6.0 to 3.6.2 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/151
* Bump poetry from 1.7.1 to 1.8.1 in /.github/workflows by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/152
* Bump black from 22.12.0 to 24.3.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/154
* Bump poetry from 1.8.1 to 1.8.2 in /.github/workflows by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/155
* Bump safety from 3.0.1 to 3.1.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/157
* Bump idna from 3.4 to 3.7 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/162
* Bump pyupgrade from 3.15.1 to 3.15.2 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/159
* Bump aiohttp from 3.9.3 to 3.9.4 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/164
* Bump flake8-builtins from 2.2.0 to 2.5.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/163
* Bump pytest from 7.4.4 to 8.1.1 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/156
* Bump pytest-cov from 4.1.0 to 5.0.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/158
* Bump dependabot/fetch-metadata from 1 to 2 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/161
* Bump release-drafter/release-drafter from 5.25.0 to 6.0.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/144
* Bump aresponses from 2.1.6 to 3.0.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/146
* Bump pip from 23.3.2 to 24.0 in /.github/workflows by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/153
* Bump safety from 3.1.0 to 3.2.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/166
* Bump pre-commit-hooks from 4.5.0 to 4.6.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/167
* Bump bandit from 1.7.7 to 1.7.8 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/168
* Bump freezegun from 1.4.0 to 1.5.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/170
* Bump pre-commit from 3.6.2 to 3.7.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/171
* Bump pytest-asyncio from 0.23.5 to 0.23.6 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/172
* Bump pytest from 8.1.1 to 8.2.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/169
* Bump freezegun from 1.4.0 to 1.5.0 in /.github/workflows by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/173
* Bump jinja2 from 3.1.3 to 3.1.4 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/174
* Bump requests from 2.31.0 to 2.32.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/175
* Bump freezegun from 1.5.0 to 1.5.1 in /.github/workflows by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/179
* Bump poetry from 1.8.2 to 1.8.3 in /.github/workflows by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/180
* Bump pytest-asyncio from 0.23.6 to 0.23.7 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/181
* Bump aiohttp from 3.9.4 to 3.9.5 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/184
* Bump pylint from 3.1.0 to 3.2.2 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/182
* Bump black from 24.3.0 to 24.4.2 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/183
* Bump pre-commit from 3.7.0 to 3.7.1 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/187
* Bump codespell from 2.2.6 to 2.3.0 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/185
* Bump pytest from 8.2.0 to 8.2.1 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/188
* Bump freezegun from 1.5.0 to 1.5.1 by @dependabot in https://github.com/DCSBL/python-frank-energie/pull/186

## New Contributors
* @berendhaan made their first contribution in https://github.com/DCSBL/python-frank-energie/pull/165

**Full Changelog**: https://github.com/DCSBL/python-frank-energie/compare/6.0.0...6.1.0

**Comment by DCSBL** [comment]
@berendhaan Please let me know if all works as expected, and I will stamp it as 'non-beta'

**Reply by berendhaan** [reply]
Are you okay with me creating the 6.1.0 release? 

How do you create the release, via the 'Draft a new release' button, or by adding a tag to a commit?

**Reply by DCSBL** [reply]
Released 6.1.0 for you, thank you for your contribution.

Feel free to suggest, merge and release small improvements. To release you can indeed:
- create a release
- give it a tag, this will be used as version
- hit 'generate release notes' and adjust when needed, I split the dependency bumps from actual changes to make it more readable.

**Comment by berendhaan** [comment]
@DCSBL If you could add the latest PR to the list above, that would be great!
Using the lib to retrieve data looks correct (for my account at least).
